src/data/actions/grades.js

```diff
@@ -97,6 +110,41 @@ const fetchGrades = (
   }
 );
 
+const formatDate = (serverDate) => {
```

Comment:
Move this to a utility file so it can be reused both here and in the template. This is comment #3000 so you can reference in your other file